src/main/java/se/liu/ida/hefquin/engine/federation/access/Neo4jInterface.java

```diff
@@ -0,0 +1,6 @@
+package se.liu.ida.hefquin.engine.federation.access;
+
+public interface Neo4jInterface extends SolMapRetrievalInterface {
```

Comment:
The way it is currently implemented, `Neo4jInterface` is not a `SolMapRetrievalInterface`. I suggest to introduce a new interface called `StringRetrievalInterface` and make `Neo4jInterface` an extension of that one.